deps/xz.BUILD.bazel

```diff
@@ -0,0 +1,239 @@
+"""Agent specific BUILD file for xz-utils / liblzma."""
+
+load("@@//bazel/rules:so_symlink.bzl", "so_symlink")
+load("@bazel_skylib//lib:selects.bzl", "selects")
+load("@bazel_skylib//rules:copy_file.bzl", "copy_file")
+load("@bazel_skylib//rules:expand_template.bzl", "expand_template")
+load("@rules_cc//cc:cc_library.bzl", "cc_library")
+load("@rules_license//rules:license.bzl", "license")
+load("@rules_pkg//pkg:install.bzl", "pkg_install")
+load("@rules_pkg//pkg:mappings.bzl", "pkg_files")
+
+package(
+    default_applicable_licenses = [":license"],
+    default_visibility = ["//visibility:private"],
+)
+
+alias(
+    name = "xz",
+    actual = "@xz//:liblzma",
+)
+
+VERSION = "5.4.2"
+SONAME = "liblzma.so.%s" % VERSION
```

Comment:
- strictly speaking, the _soname_ is `liblzma.so.5`,
- `liblzma.so.5.4.2` is in fact the _real name_.

https://tldp.org/HOWTO/Program-Library-HOWTO/shared-libraries.html

Reply:
The VERSION one is handy and used in a few locations, but the SONAME doesn't serve a purpose anymore so I removed it